tools/gen-random-bout/gen-random-bout.cpp

```diff
@@ -8,6 +8,7 @@
 //===----------------------------------------------------------------------===//
 
 #include <assert.h>
+#include <fcntl.h>
```

Comment:
Could you please change it to cassert, cstdio, cstdlib, cstring, ctime and move those up into a separate block?